### PR TITLE
Add test_claim_on_remote_revoked_sizeable_push_msat

### PR DESCRIPTION
**Pull request by ariard**
Just add a test for the sizeable push_msat branch on revoked tx in check_spend_remote_tx that I forgot in #230 

Note : may combine check_*_output_* macros into a generic one the future if there is more tests with multiplication of cases.

### CODE REVIEW

File: src/ln/channelmanager.rs [hunk 1]
```diff
@@ -7673,7 +7673,7 @@ mod tests {
 										spend_tx.input[0].witness.push(remotepubkey.serialize().to_vec());
 										txn.push(spend_tx);
 									},
-									_ => panic!("Unexpected event"),
+									_ => { if $expected == true {} else { panic!("Unexpected event")}},
```
[the remark this answers is not in the document]
Comment (ariard) [reply src/ln/channelmanager.rs]: Done, merge all into one, should have been done at #230 but was already big!

File: src/ln/channelmanager.rs [hunk 2]
```diff
@@ -7786,12 +7786,39 @@ mod tests {
...
-		let spend_txn = check_dynamic_output_p2wpkh!(nodes[1]);
+		let spend_txn = check_dynamic_output_p2wpkh!(nodes[1], false);
 		assert_eq!(spend_txn.len(), 2);
 		assert_eq!(spend_txn[0], spend_txn[1]);
 		check_spends!(spend_txn[0], node_txn[0].clone());
 	}
 
+	#[test]
+	fn test_claim_on_remote_revoked_sizeable_push_msat() {
+		// Same test as precedent, just test on remote revoked commitment tx, as per_commitment_point registration changes following you're funder/fundee and
```
[the remark this answers is not in the document]
Comment (ariard) [reply src/ln/channelmanager.rs]: Ah yes, corrected

File: src/ln/channelmanager.rs [hunk 3]
```diff
@@ -4167,6 +4167,27 @@ mod tests {
 		nodes
 	}
 
+	macro_rules! display_txn {
```
Comment (TheBlueMatt) [inline src/ln/channelmanager.rs]: Hmm, why bother adding this (it looks unused atm)? Can't we just "{:?}" the transaction itself, or did you write this and accidentally commit it?

File: src/ln/channelmanager.rs [hunk 4]
```diff
@@ -7953,10 +7972,13 @@ mod tests {
 		assert_eq!(node_txn[3].input.len(), 1);
 		check_spends!(node_txn[3], revoked_htlc_txn[0].clone());
 
-		let events = nodes[0].chan_monitor.simple_monitor.get_and_clear_pending_events();
 		// Check A's ChannelMonitor was able to generate the right spendable output descriptor
-		let spend_tx = check_static_output!(events, nodes, 1, 2, 1, 0);
-		check_spends!(spend_tx, node_txn[3].clone());
+		let spend_txn = check_spendable_outputs!(nodes[0], 1);
+		assert_eq!(spend_txn.len(), 5);
+		assert_eq!(spend_txn[1], spend_txn[3]);
```
Comment (TheBlueMatt) [inline src/ln/channelmanager.rs]: I think you meant to assert spend_txn[0] == spend_txn[2] here, too.

**Pull request closed without merging**